src/drivers/intel/cavs/mn.c

```diff
@@ -318,6 +369,8 @@ int mn_set_bclk(uint32_t dai_index, uint32_t bclk_rate,
 
 	spin_lock(mn_lock);
 
+	bclk_sources[dai_index] = MN_BCLK_SOURCE_NONE;
+
```

Comment:
why is this here? Should be in different patch related to blck?

Reply:
You are right, thanks for catching this